src/container.ts

```diff
@@ -107,16 +130,40 @@ container.register({
   apiUrl: awilix.asValue(process.env.API_URL)
 });
 
+// Notification URL pre shared key.
+const DEFAULT_PRE_SHARED_KEY = "12345";
+const preSharedKey: string =
```

Comment:
what's the scenario for a default key? I would avoid having a default key and fail the startup if no key is defined